data.sql

```diff
@@ -12,4 +12,27 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', date '2005-06-12', 1, true, -45);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', date '2005-06-07', 7, true, 20.4);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', date '1998-10-13', 3, true, 17);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (' Ditto', date '2022-05-14', 4, true, 22);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (' Ditto', date '2022-05-14', 4, true, 22);
+
+-- project for multiple tables., inserting data in owners table 
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
```

Comment:
Great job @Ngufor-emmanuella 🤝

- [ ] As stated in the project requirements, kindly insert all the data for `vets`, `specializations`, and `visits` into the data.sql file.

Reply:
thank you sorry i submitted for the wrong project.this project work is for day three which is Vet clinic database: query multiple tables thanks